utils/doclint/README.md

```diff
@@ -6,9 +6,9 @@ Puppeteer's source code.
 Doclint works in a few steps:
 
 1. Read sources in `lib/` folder, parse AST trees and extract public API
-    - note that we run DocLint on the outputted JavaScript in `lib/` rather than the source code in `src/`. We will do this until we have migrated `src/` to be exclusively TypeScript and then we can update DocLint to support TypeScript.
+   - note that we run DocLint on the outputted JavaScript in `lib/` rather than the source code in `src/`. We will do this until we have migrated `src/` to be exclusively TypeScript and then we can update DocLint to support TypeScript.
 2. Read sources in `docs/` folder, render markdown to HTML, use puppeteer to traverse the HTML
-  and extract described API
+   and extract described API
```

Comment:
GitHub won't let me make the suggestion, but this should be on the same line. Same for `1.` above.

Reply:
I've applied the change in 1f39417.